### ISSUE TITLE
npm package accidently deprecated?

### ISSUE THREAD

**Issue by ahocevar** [mapbox/vector-tile-js#52]
When installing the [vector-tile](https://npmjs.com/package/vector-tile) package, the following deprecation warning is shown:
```
npm WARN deprecated vector-tile@1.3.0: This module is now under the @mapbox namespace: install @mapbox/vector-tile instead
```
But the [@mapbox/vector-tile](https://npmjs.com/package/@mapbox/vector-tile) package appears to be something completely different.

Is it possible that the [vector-tile](https://npmjs.com/package/vector-tile) package was accidently deprecated?

**Comment by tmcw** [comment]
Thanks for the catch!

Yeah, so... what I think happened was that we had both `vector-tile-js` and `vector-tile` repos, both with package.json files, neither package.json was `private`, and my audit script put `mapbox/vector-tile` into the namespace first, since it sorted alphabetically, and didn't account for the possibility of two same-named repositories.

I just un-deprecated `vector-tile`, and I think the most reasonable approach would be to:

* keep `@mapbox/vector-tile` as https://github.com/mapbox/vector-tile
* move `vector-tile-js` to `@mapbox/vector-tile-js`

Does this sound all right @springmeyer @jfirebaugh ?

**Comment by jfirebaugh** [comment]
IMO, the JS package should be `@mapbox/vector-tile`, the C++ header library should be `@mapbox/vector-tile-hpp`.

**Comment by tmcw** [comment]
Ideally, that's what we'd do, but given the current rules around unpublishing and given that `@mapbox/vector-tile` is already published as the C++ version, I don't think that's the best real-world option.

**Comment by jfirebaugh** [comment]
`@mapbox/vector-tile` is currently published at `1.0.0-alpha.1` and has no dependents. I don't think it would be a huge deal to publish the JS package over it at `1.3.0`.

**Comment by springmeyer** [comment]
Please hold on republishing the JS package over the C++ package. This is possible but we have one @mapbox internal module that this would break badly because it uses `^1.0.0`. I'll work on retagging that modules to pin to `1.0.0-alpha.1` to avoid the problem. Then I'll ping back here.

**Comment by jlaine** [comment]
Any news on this? Presently it's impossible to install mapbox-gl-js without triggering deprecation warnings

**Comment by jfirebaugh** [comment]
[`@mapbox/vector-tile`](https://www.npmjs.com/package/@mapbox/vector-tile) has been published at 1.3.0. Upgrade to this version to silence the deprecation warnings.
